### ISSUE TITLE
BUG: App UI never appears on Windows 10

### ISSUE THREAD

**Issue by f0ff886f** [dbgate/dbgate#86]
**Describe the bug**
When I launch dbgate, the UI never shows up. I have a taskbar entry, and processes in the background. When I close the taskbar entry, the processes stay alive.

**To Reproduce**
(This happens directly after installation or if you kill all processes and start from 1.)
1. Click Start -> dbgate shortcut
2. Observe the taskbar:
![image](https://user-images.githubusercontent.com/528239/115721711-9a9b2e00-a37e-11eb-9131-4085f4cb7f78.png)
3. Observe no UI window showing up at all
4. Observe process list:
![image](https://user-images.githubusercontent.com/528239/115721867-bbfc1a00-a37e-11eb-8c6f-fc119dce8bed.png)
5. Right click on Taskbar entry, close.
6. Observe process list:
![image](https://user-images.githubusercontent.com/528239/115721985-d635f800-a37e-11eb-9721-ed9d8fba5d53.png)

**Expected behavior**
The app UI to show up.

**Screenshots**
See repro steps.

**Version Information (please complete the following information):**
 - OS: Windows 10 Pro 20H2
 - App: Version 4.1.9
 - Install source: EXE Installer dbgate-windows-4.1.9.exe
 - Type: Application
 - Database engine: Didn't get that far

**Additional context**
 - NVIDIA GPU installed, driver 465.89

**Comment by janproch** [comment]
It looks like electron problem. I tried it now on two windows 10 machines and it works

Could you please try to start dbgate.exe from console and copy console output?

**Comment by jeremywww** [comment]
`15:40:55.135 > Checking for update
(node:29976) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
(node:29976) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
Calling init controller for controller /connections
Calling init controller for controller /scheduler
Cleaning directory C:\Users\username\dbgate-data\run
15:40:56.447 > Update for version 4.1.9 is not available (latest version: 4.1.9, downgrade is disallowed).
15:40:56.452 > checkForUpdatesAndNotify called, downloadPromise is null
events.js:292
      throw er; // Unhandled 'error' event
      ^
 
Error: listen EACCES: permission denied 0.0.0.0:53911
    at Server.setupListenHandle [as _listen2] (net.js:1296:21)
    at listenInCluster (net.js:1361:12)
    at Server.listen (net.js:1447:7)
    at C:\Users\username\AppData\Local\Programs\dbgate\resources\app.asar\packages\api\dist\bundle.js:91:126042
    at p (C:\Users\username\AppData\Local\Programs\dbgate\resources\app.asar\packages\api\dist\bundle.js:358:25398)
    at Socket.<anonymous> (C:\Users\username\AppData\Local\Programs\dbgate\resources\app.asar\packages\api\dist\bundle.js:358:25345)
    at Socket.emit (events.js:315:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on Server instance at:
    at emitErrorNT (net.js:1340:8)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  code: 'EACCES',
  errno: 'EACCES',
  syscall: 'listen',
  address: '0.0.0.0',
  port: 53911
}`

**Comment by chocolatkey** [comment]
Just tried and had the same issue. As soon as I saw the console logs, I knew what the issue was - HyperV's reserved port ranges
![image](https://user-images.githubusercontent.com/13407784/115779530-f52b8e80-a36c-11eb-9186-cb8ac8bb8f91.png)

**Comment by f0ff886f** [comment]
Just to complete the circle, yes, exact same symptom as everyone else:

```Error: listen EACCES: permission denied 0.0.0.0:53911
    at Server.setupListenHandle [as _listen2] (net.js:1296:21)
    at listenInCluster (net.js:1361:12)
    at Server.listen (net.js:1447:7)```

However, I checked in CurrPorts and nothing was listening to that range, so I don't have the same problem as @chocolatkey.

I disconnected from my VPN which has been causing me issues with opening ports lately, and there we go! DBGate now opens just fine and is able to bind to the port.

I did not get any Firewall popup dialogs while it was opening, although I am explicitly adding rules to the firewall and it still won't launch, so this may be something funky with my firewall settings (just using Windows Firewall but my VPN Client (Cisco AnyConnect) seems to behave oddly).

**Comment by janproch** [comment]
```
  netsh int ip show excludedportrange protocol=tcp
```

**Comment by janproch** [comment]
Should be fixed in BETA release 4.1.10-beta.3

I have exchanged library for finding free port (which also uses random port), seems to be more robust

https://github.com/dbgate/dbgate/releases

**Comment by janproch** [comment]
fixed in version 4.1.10

**Comment by kleinstein** [comment]
I have installed version : [v5.2.7](https://github.com/dbgate/dbgate/releases/tag/v5.2.7)
And when i start DBGate, I can see 4 Process with the name DBGate.exe but i don't see any UI...
